pkg/tfgen/generate_go_utilities.go

```diff
@@ -0,0 +1,45 @@
+package tfgen
+
+const goUtilitiesFile = `
```

Comment:
I guess we don't have an "overlays" mechanism for our Go code generator?  Surprised we can't just read this file as a `.go` file off disk and place it in the desired location.